bin/README.md

```diff
@@ -0,0 +1,26 @@
+## Bin
+
+This directory contains a number of scripts for running pathway parameter advising, updating reference pathways, and installing PGD.
+
+Most of these scripts must be run inside the `bin` directory. 
+
+## Files
+
+`bin/runPPA.sh` runs pathway parameter advising on any set of sif or edgelist networks.
+It will attempt to install the pgd library and setup the reactome pathway graphlets automatically. 
+The following commands will run `runPPA.sh` with the included `Wnt` and `Prolactin` datasets from the `scripts` directory:
```

Comment:
In general, I recommend having this information in one place, either the main readme or the bin readme.  That gives us less maintenance overhead and helps keep instructions from diverging.  If we want the usage to be here, then the main readme can refer users to the bin readme for a descriptions of the scripts and their functions, telling users that is the main way to run PPA.

This suggestion applies for all the scripts here.

Reply:
I decided to move all of the information to the main readme. The bin readme now only contains a short description and a list of files.